docs/content/doc/usage/email-setup.en-us.md

```diff
@@ -15,19 +15,31 @@ menu:
 
 # Email setup
 
-- To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
+To use Gitea's built-in Email support, update the `app.ini` config file [mailer] section:
 
+- Sendmail version - Use the operating system’s sendmail command instead of SMTP. This is common on linux servers.
```

Comment:
```suggestion
## Sendmail version 
Use the operating system’s sendmail command instead of SMTP. This is common on linux servers.  
Note: For use in the official gitea docker image, please configure with the SMTP version.
```

Reply:
@sapk thanks.